File: contractstudy/src/test/java/test/contractstudy/extractor/common/StaticImportCollector/java/testData/StaticImportAllStatic.java
package test.contractstudy.extractor.common.StaticImportCollector.java.testData;

import static java.lang.Math.PI;
import static java.lang.System.out;

public class StaticImportAllStatic {

  public boolean test(boolean flag) {
    return !flag;
  }

  public double sum(int num1, double num2) {
    return num1 + num2;
  }

  public double compute(int area1) {
    return sum(area1, PI);
  }

  public void print(String text) {
    out.println(text);
  }
}

File: contractstudy/src/test/java/test/contractstudy/extractor/common/StaticImportCollector/kotlin/testData/StaticImportAllStatic.kt
package test.contractstudy.extractor.common.StaticImportCollector.kotlin.testData


class StaticImportAllStatic {
    fun test(flag: Boolean): Boolean {
        return !flag
    }

    private fun sum(num1: Int, num2: Double): Double {
        return num1 + num2
    }

    fun compute(area1: Int): Double {
        return sum(area1, Math.PI)
    }

    fun print(text: String?) {
        println(text)
    }
}